CMake/CMakeTestISPCCompiler.cmake

```diff
@@ -0,0 +1,2 @@
+# For now just do nothing in here
+set(CMAKE_ISPC_COMPILER_WORKS 1 CACHE INTERNAL "")
```

Comment:
Would that make sense to at least check here that `find_program` in `CMake/CMakeDetermineISPCCompiler.cmake` worked?

Reply:
missing new line at end of file